processor/patch.go

```diff
@@ -26,7 +26,7 @@ func (p *Processor) patchFile(fset *token.FileSet, file *ast.File, patches ...pa
 		buf.Reset()
 
 		buf.WriteString("\n")
-		if err := format.Node(&buf, fset, patch.stmts); err != nil {
+		if err = format.Node(&buf, fset, patch.stmts); err != nil {
```

Comment:
new lexical scope is good usually
```suggestion
		if err := format.Node(&buf, fset, patch.stmts); err != nil {
```

Reply:
I prefer to avoid shadowing variables. Ideally, I should prepend the `err` declaration. However, it's not a significant issue.